src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/mate";
+    private static final String DB_USER = "root";
+    private static final String DB_PASSWORD = "Qwerty123";
```

Comment:
Storing database credentials in the source code is a security risk. Consider using environment variables or a configuration file to manage credentials securely.